lib/sshutils/scp/scp.go

```diff
@@ -139,6 +144,18 @@ func (cmd *Command) sendDir(r *reader, ch io.ReadWriter, fi os.FileInfo, path st
 }
 
 func (cmd *Command) sendFile(r *reader, ch io.ReadWriter, fi os.FileInfo, path string) error {
+	// log audit event:
+	if cmd.AuditLog != nil {
```

Comment:
suggestion- instead of checking for nil all the time, may be set dummy `nop` audit logger implementation